src/Logary.Facade/Facade.fs

```diff
@@ -259,31 +426,6 @@ module internal Formatting =
     let formatInstant (utcTicks : int64) =
       (DateTimeOffset(utcTicks, TimeSpan.Zero).ToString("o")) + ": "
 
-    let formatValue fields = function
```

Comment:
Also, to reduce parser complexity, I dropped the feature that returned the index. It's hard with support for `{{` which is better for the user who wants braces in their output